api/v1alpha1/perconaservermysql_types.go

```diff
@@ -361,6 +371,10 @@ func (cr *PerconaServerMySQL) OrchestratorSpec() *OrchestratorSpec {
 	return &cr.Spec.Orchestrator
 }
 
+func (cr *PerconaServerMySQL) HAProxyEnabled() bool {
```

Comment:
add the following line ` //+kubebuilder:printcolumn:name="Haproxy",type=string,JSONPath=".status.haproxy.ready"` after` // +kubebuilder:printcolumn:name="Orchestrator",type=string,JSONPath=".status.orchestrator.ready"`

we do not have haproxy in `kubectl get ps` output